lib/ring_logger/server.ex

```diff
@@ -285,4 +319,38 @@ defmodule RingLogger.Server do
       }
     end)
   end
+
+  defp load_persist_path(state) do
+    case Persistence.load(state.persist_path) do
+      logs when is_list(logs) ->
+        state =
+          Enum.reduce(logs, state, fn log_entry, state ->
+            insert_log(state, log_entry)
+          end)
+
+        %{state | index: Enum.count(logs)}
+
+      {:error, :corrupted} ->
+        timestamp = :os.system_time(:microsecond)
```

Comment:
This is from Logger here https://github.com/elixir-lang/elixir/blob/main/lib/logger/lib/logger/utils.ex#L175

Unsure if we should even use this function so I copied it in. We might also want local time instead of UTC?